evalml/tests/component_tests/test_estimators.py

```diff
@@ -98,23 +98,28 @@ def test_all_estimators_check_fit_input_type_regression(data_type, X_y_regressio
         component.predict(X)
 
 
-def test_estimator_predict_output_type(X_y_binary, helper_functions):
+def test_estimator_predict_output_type(X_y_binary, ts_data, helper_functions):
     X_np, y_np = X_y_binary
     assert isinstance(X_np, np.ndarray)
     assert isinstance(y_np, np.ndarray)
+
     y_list = list(y_np)
     X_df_no_col_names = pd.DataFrame(X_np)
     range_index = pd.RangeIndex(start=0, stop=X_np.shape[1], step=1)
     X_df_with_col_names = pd.DataFrame(X_np, columns=['x' + str(i) for i in range(X_np.shape[1])])
     y_series_no_name = pd.Series(y_np)
     y_series_with_name = pd.Series(y_np, name='target')
-    datatype_combos = [(X_np, y_np, range_index, np.unique(y_np)),
-                       (X_np, y_list, range_index, np.unique(y_np)),
-                       (X_df_no_col_names, y_series_no_name, range_index, y_series_no_name.unique()),
-                       (X_df_with_col_names, y_series_with_name, X_df_with_col_names.columns, y_series_with_name.unique())]
+    X_df_no_col_names_ts = pd.DataFrame(data=X_df_no_col_names.values, columns=X_df_no_col_names.columns,
+                                        index=pd.date_range(start='1/1/2018', periods=X_df_no_col_names.shape[0]))
+
+    datatype_combos = [(X_np, y_np, range_index, np.unique(y_np), False),
+                       (X_np, y_list, range_index, np.unique(y_np), False),
+                       (X_df_no_col_names, y_series_no_name, range_index, y_series_no_name.unique(), False),
+                       (X_df_with_col_names, y_series_with_name, X_df_with_col_names.columns, y_series_with_name.unique(), False),
+                       (X_df_no_col_names_ts, y_series_no_name, range_index, y_series_no_name.unique(), True)]
 
     for component_class in _all_estimators_used_in_search():
-        for X, y, X_cols_expected, y_cols_expected in datatype_combos:
+        for X, y, X_cols_expected, y_cols_expected, time_series in datatype_combos:
```

Comment:
is `time_series` not used in the for loop? If not, can we remove it?

Reply:
That's weird, I had previously changed this to include it with specific cases for time series...I think it might have been overwritten in the last merge to main, let me add that back!